src/main/java/org/apache/commons/lang3/RandomStringUtils.java

```diff
@@ -95,6 +95,21 @@ public static String randomAscii(final int count) {
     public static String randomAlphabetic(final int count) {
         return random(count, true, false);
     }
+
+    /**
+     * <p>Creates a random string whose length is the number of characters
+     * specified.</p>
```

Comment:
Dosn't exactly describe what the method does. More like `Creates a random string of length between minLength and maxLength`.